.pre-commit-config.yaml

```diff
@@ -26,8 +26,8 @@ repos:
       - types-filelock
       - types-setuptools
       - arviz
-      - aesara==2.4.0
-      - aeppl==0.0.26
```

Comment:
Oh great, even more places for the dependencies xD

Reply:
You can't imagine how delighted I was to learn that the greatness of YAML required me to add the dependencies in yet another place! `</sarcasm>`